### PR TITLE
New group lead enter contact address

### PR DESCRIPTION
**Pull request by mycp98**
### Description of Work carried through

Brief description of what/why/how.

### CODE REVIEW

File: app/views/changeGroupLead/NewGroupLeadEnterContactAddressView.scala.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 @(form: Form[_], organisationName: String, mode: Mode)(implicit request: Request[_], messages: Messages)
 
-@input(formName: String, key: String, id: String) = @{
+@input(formName: String, key: String) = @{
```
Comment (liampan) [inline app/views/changeGroupLead/NewGroupLeadEnterContactAddressView.scala.html:32]: need to override the id to match the form name field

File: app/views/changeGroupLead/NewGroupLeadEnterContactAddressView.scala.html [hunk 2]
```diff
@@ -51,12 +51,12 @@
 
         @paragraphBody(messages("newGroupLeadEnterContactAddress.paragraph"))
 
-        @input("addressLine1", "newGroupLeadEnterContactAddress.addressLine1.label", "address-line-1")
-        @input("addressLine2", "newGroupLeadEnterContactAddress.addressLine2.label", "address-line-2")
-        @input("addressLine3", "newGroupLeadEnterContactAddress.addressLine3.label", "address-line-3")
-        @input("addressLine4", "newGroupLeadEnterContactAddress.addressLine4.label", "address-line-4")
-        @input("postalCode", "newGroupLeadEnterContactAddress.postalCode.label", "postal-code")
-        @input("countryCode", "newGroupLeadEnterContactAddress.countryCode.label", "country-code")
+        @input("addressLine1", "newGroupLeadEnterContactAddress.addressLine1.label")
```
Comment (liampan) [inline app/views/changeGroupLead/NewGroupLeadEnterContactAddressView.scala.html:54]: can these use the object variables

**Review by liampan — APPROVED** [review]
Will need to edit the id's for the error link to work

File: app/views/changeGroupLead/NewGroupLeadEnterContactAddressView.scala.html (hunk 2 above)
Comment (mauro-fanton) [reply app/views/changeGroupLead/NewGroupLeadEnterContactAddressView.scala.html:54]: yes, I agree.

**Pull request merged**